apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.IntegrationTests/UploadTest.cs

```diff
@@ -134,8 +190,8 @@ public void UploadAvro_IgnoresSchema()
             }
             var rows = client.GetTable(tableReference).ListRows().ToList();
             Assert.Equal(1, rows.Count);
-            Assert.Equal(100, (long) rows[0]["re"]);
-            Assert.Equal(200, (long) rows[0]["im"]);
+            Assert.Equal(100, (long)rows[0]["re"]);
```

Comment:
I'm not sure what's been automatically formatting the casts, but it would be good if you could turn it off. (I don't much mind whether there's a space or not, but it's better not to have unrelated changes in the commit.)

Don't worry about undoing these ones - let's just focus on avoiding them in the future.

Reply:
That would be Edit -> Advanced -> Format Document
Won't do it again.